### DISCUSSION TITLE
Support for mysql

### DISCUSSION THREAD

**Discussion by maboelnour** [palkan/logidze#177] [Ideas]
Are there any plans to support MySQL? if not can we start a discussion/brainstorm about it?

**Comment by palkan** [comment]
We don't have plans to work on MySQL support in the nearest future, but we're open to contributions in this direction.

As far as I know, MySQL supports [triggers](https://dev.mysql.com/doc/refman/8.0/en/triggers.html) and has a built-in [JSON type](https://dev.mysql.com/doc/refman/8.0/en/json.html), which could be used instead of JSONB.

I'd suggest to start with building a PoC, running some benchmarks and decide whether it worth an effort to continue.
